<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>699</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CarpView</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="lstDomains">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tblEntries"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblZeroMqState">
      <property name="text">
       <string>ZeroMQ: Disabled</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Raw"/>
    <addaction name="actionLoad_Domain_File"/>
    <addaction name="actionLoad_aggregate_domain_file"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuZeroMQ">
    <property name="title">
     <string>ZeroMQ</string>
    </property>
    <addaction name="actionZeroMqToggle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuZeroMQ"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Raw">
   <property name="text">
    <string>Open Raw</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Domain_File">
   <property name="text">
    <string>Load Domain File</string>
   </property>
  </action>
  <action name="actionLoad_aggregate_domain_file">
   <property name="text">
    <string>Load aggregate domain file</string>
   </property>
  </action>
  <action name="actionZeroMqToggle">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
